cypress/integration/logistics/materialentnahme_partial_HU.js

```diff
@@ -0,0 +1,244 @@
+      Object.assign(new DiscountSchema(), discountSchemaJson)
+        .setName(discountSchemaName)
+        .apply();
+    });
+  });
+  it('create product from 24_Gebinde category in order to use it for packing material for transport unit', function() {
+    // eslint-disable-next-line
+    Builder.createProductWithPriceUsingExistingCategory(priceListName, productForPackingMaterialTU, productForPackingMaterialTU, productType, "24_Gebinde");
+  });
+  it('create product from 24_Gebinde category in order to use it for packing material for loading unit', function() {
+    Builder.createProductWithPriceUsingExistingCategory(
+      priceListName,
+      productForPackingMaterialLU,
+      productForPackingMaterialLU,
+      productType,
+      '24_Gebinde'
+    );
+  });
+  it('create packing material for handling unit - Transport unit', function() {
+    cy.fixture('product/packing_material.json').then(packingMaterialJson => {
+      Object.assign(new PackingMaterial(), packingMaterialJson)
+        .setName(packingMaterialForTU)
+        .setProduct(productForPackingMaterialTU)
+        .setLength('0')
+        .setWidth('0')
+        .setHeight('0')
+        .apply();
+    });
+  });
+  it('create packing material for handling unit - Load unit', function() {
+    cy.fixture('product/packing_material.json').then(packingMaterialJson => {
+      Object.assign(new PackingMaterial(), packingMaterialJson)
+        .setName(packingMaterialForLU)
+        .setProduct(productForPackingMaterialLU)
+        .setLength('0')
+        .setWidth('0')
+        .setHeight('0')
+        .apply();
+    });
+  });
+  it('create packing instruction for handling unit - Transport unit', function() {
+    cy.fixture('product/packing_instructions.json').then(packingInstructionsJson => {
+      Object.assign(new PackingInstructions(), packingInstructionsJson)
+        .setName(packingInstructionsNameForTU)
+        .apply();
+    });
+  });
+  it('create packing instruction for handling unit - Load unit', function() {
+    cy.fixture('product/packing_instructions.json').then(packingInstructionsJson => {
+      Object.assign(new PackingInstructions(), packingInstructionsJson)
+        .setName(packingInstructionsNameForLU)
+        .apply();
+    });
+  });
+  it('create packing instruction version for handling unit - Transport unit; with packgut - quantity=0.00 and packmittel', function() {
+    cy.fixture('product/packing_instructions_version.json').then(pivJson => {
+      Object.assign(new PackingInstructionsVersion(), pivJson)
+        .setName(packingInstructionsVersionForTU)
+        .setPackingInstructions(packingInstructionsNameForTU)
+        .setPackingMaterial(packingMaterialForTU)
+        .apply();
+    });
+    /**For some reasons,the quantity for the packgut cannot be added in advanced edit mode, only like below */
+    cy.selectTab('M_HU_PI_Item');
+    cy.selectNthRow(1)
+      .find('.Quantity')
+      .dblclick({ force: true })
```

Comment:
can we do clicks and typing without `force`?

Reply:
I took them out for now, but randomly this fails and that's why I have added the force: true. I have the feeling that we will need it in the future, but for the mom let's leave it without.